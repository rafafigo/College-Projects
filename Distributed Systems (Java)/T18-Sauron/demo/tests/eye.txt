# Capture Begin!
zzz 1000
car, 87JX26
car, 40IY72
person, 8145431631784073314
car, 00UX85
person, 2387367386175552764
person, 8145431631784073314
car, 83VO32
car, 35LS08
car, 54ZE32
person, 5036918424659964097
person, 5036918424659964097
person, 8145431631784073314
car, 85MJ89
car, 34GS92
car, 41YT75
car, 67NA36
person, 4054597058007472827
car, 35HL80
car, 86SF44
person, 384119406722942470
person, 1712043095926672962
car, 99TW22
car, 24WL14
car, 29MB19
car, 12VO39
person, 4054597058007472827
person, 542280985820332376
car, 93WD04
person, 7891683966282422742
person, 8337834082749331789
car, 37OC46
car, 22RQ43
car, 77XA92
car, 31HH70
car, 14HE18
car, 46BH13
person, 384119406722942470
person, 8280668941620679147
person, 7891683966282422742
car, 78SO33
car, 67LQ96
person, 4614612542812256924
person, 2520536790426462833
person, 8280668941620679147
person, 8006551120741991985
car, 17CE76
car, 89TX46
car, 83KW11
car, 76BR80
person, 9011418505957728885
car, 50ES61
car, 38AQ95
car, 89IW87
car, 59KL74
person, 1631290688234576111
car, 86FX86
car, 03ZW61
person, 6329558186210721135
car, 88KS64
car, 31OA31
# Processing Data Pause a Little Bit!
zzz 1000
person, 4022072066264733433
person, 6503551766859413663
person, 8347515276177464306
car, 03XQ77
person, 2355884487527032718
car, 05TF38
person, 6097793726902230637
car, 48IY46
car, 89NU27
person, 4591694522614271560
car, 66XV70
person, 7480730864576976876
person, 2897035744902635202
car, 01RE71
car, 57OR31
person, 1631290688234576111
car, 77VI18
person, 2204663528393598007
car, 23IV00
car, 91SQ56
car, 15TQ23
car, 24EP18
car, 50TD28
car, 88BB07
person, 8481586198431302042
car, 77SC75
person, 1331701354219002597
person, 2355884487527032718
person, 3837570317995403263
person, 3981196982013561507
# Processing Data Pause a Little Bit!
zzz 1000
person, 7330987785287994978
car, 60MO32
car, 65TV31
car, 13ED24
car, 02YC99
car, 34YE82
person, 8481586198431302042
car, 66CV37
person, 8152074906356500099
person, 3756467543486240378
car, 64VJ18
car, 92KP68
person, 1072661629839433060
car, 20GL60
person, 2204663528393598007
person, 690678069381844092
car, 29JB61
car, 51CQ45
car, 08PZ49
car, 56UM93
person, 7891683966282422742
person, 4591694522614271560
person, 3194024464456255691
car, 09JG47
car, 63RR72
car, 19VB59
car, 71GM79
person, 1894397034008575033
person, 8199602241174479351
car, 26VR79
# Processing Data Pause a Little Bit!
zzz 1000
person, 8094863303557697360
person, 8852714074961603492
person, 5511331047962184539
car, 78SY99
car, 13ZB63
car, 36DB59
person, 4942632595252575521
car, 06CH46
person, 7787957968999765827
person, 1576982545850284213
car, 34ES21
person, 2872388898991369396
person, 6809400658775366916
person, 1331701354219002597
person, 7165514826495573927
person, 2802667763425121180
car, 27BB47
person, 5925445965795677759
person, 2103667955066563612
car, 51SH36
car, 96OZ12
car, 92AF96
person, 6097793726902230637
car, 43HD01
car, 81OD93
person, 5948900265997132539
car, 96FW33
person, 582614740886978115
car, 50FF44
car, 90ZC72
car, 59IE44
person, 5530634268875675736
car, 66IP27
car, 30MB28
car, 34ER77
person, 5240847274067836672
person, 8282742879714770567
person, 158391092857569395
person, 8676295962432333135
car, 35QG62
car, 92MC10
person, 8063729026100868090
car, 90LT68
person, 8481586198431302042
car, 04QM11
person, 5812885514130259966
car, 67SK07
car, 62YA92
car, 16UM22
car, 12EG80
person, 5036918424659964097
car, 90UW79
car, 78UL47
car, 86KK31
car, 03VZ92
person, 6099929353537503731
car, 28KS35
car, 83JC25
person, 3661888204889812610
person, 7169874765878845624
car, 08JT89
person, 7689455615157167744
person, 6887090144474792671
car, 51JZ80
person, 1118571451336250168
car, 86MC08
car, 99UO85
person, 5609175688942627440
person, 9185939793770365155
# Processing Data Pause a Little Bit!
zzz 1000
car, 65FW83
car, 77AJ32
person, 4474646177462363886
car, 81ZG63
person, 3300491054648848462
car, 47AE48
person, 8588196393462679247
car, 92HV44
car, 60FM99
person, 8676295962432333135
person, 1724565858515548295
# Capture End!
