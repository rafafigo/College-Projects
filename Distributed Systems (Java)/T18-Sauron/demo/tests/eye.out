EyeApp
Received 5 arguments
arg[0] = localhost
arg[1] = 8080
arg[2] = Tagus
arg[3] = 30
arg[4] = 30
